package.json

```diff
@@ -862,6 +901,30 @@
             "scope": "window",
             "default": "https://docs.runme.dev",
             "markdownDescription": "Documentation Base URL"
+          },
+          "runme.features.escalate": {
```

Comment:
I'd remove all these and generically handle settings lookup inside the code `runme.features.<feature-key` without publishing details here in `package.json` as a soft-overwrite for devs, not users.